server/components/users/user-controllers.js

```diff
@@ -0,0 +1,43 @@
+const User = require("./user-model");
+
+exports.getAllUsers = async (req, res) => {
+	try {
+		const users = await User.find({});
+
+		if (users.length > 0) {
+			res.status(200).json({users});
+		} else {
+			res.status(200).json({message: "No users yet!"});
+		}
+	} catch (error) {
+		res.status(500).json({error});
+	}
+};
+
+exports.registerUser = async (req, res) => {
+	try {
+		const newUser = await new User({
+			name: req.body.name,
+			email: req.body.email,
+			passwordHash: req.body.password
+		}).save();
+
+		res.status(201).json({newUser});
+	} catch (error) {
+		res.status(500).json({error});
+	}
+};
+
+exports.loginUser = async (req, res) => {
```

Comment:
Note for the future: I've renamed this to `loginUser` (without the `Fake` at the end), but this will do nothing until we integrate something like Passport.js or Firebase Auth into the project.